tests/dummy/app/controllers/besluit-sample.ts

```diff
@@ -355,7 +360,177 @@ export default class BesluitSampleController extends Controller {
         onlyArticleSpecialName: true,
       },
       documentValidation: {
-        documentShape: getShapeOfDocumentType('decision'),
+        actions: [
+          {
+            shaclRule:
+              'https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-title-validation',
+            action: (controller: SayController) =>
+              insertTitleAtCursor(controller, this.intl),
+            buttonTitle: 'Insert title',
+          },
+          {
+            shaclRule:
+              'https://data.vlaanderen.be/shacl/besluit-publicatie#besluit-description-validation',
+            action: (controller: SayController) =>
+              insertDescriptionAtCursor(controller, this.intl),
```

Comment:
soooo. yeah this simply doesn't work...
The problem is that the rule is checking for exactly 1 description, but the "fix" always inserts a new one. So if you have 2 (easy to do by just copy-pasting, and changing the content of 1 of them so it generates 2 different triples), it still suggests inserting one more...

The setup is simply not granular enough I think... 
Unless there's an easy fix we might have to shelve this ticket and look a this again later